Content.Shared/Interaction/Events/ContactInteractionEvent.cs

```diff
@@ -9,9 +9,11 @@ namespace Content.Shared.Interaction.Events;
 public sealed class ContactInteractionEvent : HandledEntityEventArgs
 {
     public readonly EntityUid Other;
+    public bool HandContact;
 
```

Comment:
2 things
1. Please document your fields as it's not intuitive what this means at all.
2. Is every single subscription using it now? If so it would be better to just not raise the event at all and leave comments.

Reply:
That's very fair. I think it was also used for diseases at one point, but we can worry about that when diseases are added again one day. I've just removed those instances where we don't want to raise the event for now.